test/lit/binary/stacky-eh-legacy.test

```diff
@@ -35,15 +35,15 @@
 ;; The fixup will hoist the 'pop' and create another local to store it right
 ;; after 'catch'.
 
-;; RUN: wasm-opt -all %s.wasm -S -o - | filecheck %s
+;; RUN: wasm-opt %s.wasm -S -o - | filecheck %s
```

Comment:
How does this command not error in the validator, as the wasm uses EH?

Reply:
Oh, the features section? Which is the binary change I guess.

Reply:
I added a target features section enabling EH to the binary, so this test now reflects what happens in practice when Emscripten first parses a binary.